### PR TITLE
[Term Entry] Pandas DataFrame: .sort_values()

### PR DESCRIPTION
**Pull request by alespros**
### Description

Closes [#1575](https://github.com/Codecademy/docs/issues/1575)

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist
- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@alespros All set with this first review. It's mostly small formatting edits

File: content/pandas/concepts/dataframe/terms/sortvalues/sortvalues.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+Title: '.sort_values()'
+Description: 'Sorts values in a DataFrame based on the specified column or columns.'
```
[the remark this answers is not in the document]
Comment (alespros) [reply content/pandas/concepts/dataframe/terms/sortvalues/sortvalues.md:3]: I think this option would be more descriptive, but I can use the suggested change, too.
```suggestion
Description: 'Sorts values in a DataFrame by the specified column or row.'
```

File: content/pandas/concepts/dataframe/terms/sortvalues/sortvalues.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+The **`.sort_values()`** function sorts values in a [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) along the selected axis and returns a [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) with sorted values or None.
+
+## Syntax
+
+```pseudo
+dataframevalue.sort_values(by, *, axis=0, ascending=True, inplace=False, kind='quicksort', na_position='last', ignore_index=False, key=None)
+```
+
+`.sort_values()` uses the following parameters:
+- `by`: `Name` or `list of names` to sort by.
```
[the remark this answers is not in the document]
Comment (alespros) [reply content/pandas/concepts/dataframe/terms/sortvalues/sortvalues.md:24]:
```suggestion
- `by`: A label or list of labels to sort by (levels can also specified if applicable)..
```

File: content/pandas/concepts/dataframe/terms/sortvalues/sortvalues.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (caupolicandiaz) [reply content/pandas/concepts/dataframe/terms/sortvalues/sortvalues.md:3]:
```suggestion
Description: 'Sorts values in a DataFrame based on a selection of columns or rows.'
```

[the remark this answers is not in the document]
Comment (caupolicandiaz) [reply content/pandas/concepts/dataframe/terms/sortvalues/sortvalues.md:3]: @alespros My edit is not as concise as I'd like, but I think it's important to detail that the sorting is not limited to one label in either direction.

**Review by caupolicandiaz — APPROVED** [review]
@alespros Looks good, this is ready for the next review

**Review by SSwiniarski — APPROVED** [review]
@alespros, @caupolicandiaz To me it looks good to merge.

**Comment by KTom101** [comment]
@alespros Congrats on the [.sort_values() entry](https://www.codecademy.com/resources/docs/pandas/dataframe/sortvalues)! 🎉 Please feel free to look to see if there is another unclaimed [issue](https://github.com/Codecademy/docs/issues) you want to work on or let us know if there is something currently not on Codecademy Docs you want to add!

Screen Shot 2023-02-11 at 2 24 08 PM

**Pull request merged**